<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_novaVenda</class>
 <widget class="QDialog" name="window_novaVenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nova Venda</string>
  </property>
  <widget class="QLineEdit" name="txt_veQuantidade">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_veCodigo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_veProdutos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>81</y>
     <width>691</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantidade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CÃ³digo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor Total:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_veExcluirProduto">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>330</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluir Produto</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_veEditarProduto">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar produto</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_veFinalizarVenda">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Finilizar Venda</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_vePesquisar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_veTotalVenda">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_veQuantidade</tabstop>
  <tabstop>btn_vePesquisar</tabstop>
  <tabstop>txt_veCodigo</tabstop>
  <tabstop>tw_veProdutos</tabstop>
  <tabstop>btn_veExcluirProduto</tabstop>
  <tabstop>btn_veEditarProduto</tabstop>
  <tabstop>btn_veFinalizarVenda</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
